src/translations/en.json

```diff
@@ -22,6 +22,7 @@
     "artist_toptracks": "Top tracks",
     "artists": "Artists",
     "browse": "Browse",
+    "builtin_playlists": "Playlist Settings",
```

Comment:
Are you sure this is the correct translation ? Shouldn't this be "Builtin playlists" ?

Reply:
I was trying to align with the other headings - Generic Settings, Advanced Settings, AirPlay Specific Settings but I do note we aren't that standardised

![image](https://github.com/user-attachments/assets/6d169bcc-973f-4855-9165-2e0c5cb8bdb8)

Maybe `Builtin Playlist Settings`?